tests/e2e/utils/fetch-page-content.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Fetch markup for any given URL in the context of Puppateer
```

Comment:
Typo, but also we use this verb form for docblocks, etc (I believe core does the same)
```suggestion
 * Fetches markup for any given URL in the context of Puppeteer
```